### PR TITLE
Schedule_ms10335

### PR DESCRIPTION
**Pull request by ms10335**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @ms10335

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ms10335 for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @ms10335

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ms10335 for on-time delivery

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+void schedule(std::function<void()> invokeFun, std::chrono::seconds duration) {
+    auto now = std::chrono::system_clock::now();
+    auto end = std::chrono::system_clock::now();
+
+    while (true) {
+        if (end >= (now + duration)) {
+            break;
+        }
+        end = std::chrono::system_clock::now();
+    }
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:15]: Use std::this_thread::sleeep for or sleep until instead loop. Do this in rest of functions

File: homework/schedule/schedule.hpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    //wywoluje funkcje
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:16]: Avoid polish comment. Also this comment is redundat

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**